src/Drupal/Bootstrap.php

```diff
@@ -74,7 +74,7 @@ public function register(): void
         }, $profiles);
         $this->extensionDiscovery->setProfileDirectories($profile_directories);
 
-        $this->moduleData = $this->extensionDiscovery->scan('module');
+        $this->moduleData = array_merge($this->extensionDiscovery->scan('module'), $profiles);
```

Comment:
If we want to rename the variable or method name to `extensionData` or  `addExtensionNamespaces` feel free to do that before commit. I just wanted to show the fix here. :)

Reply:
I think this is fine. Technically themes are extensions as well. But profiles are more like "special" modules anyways.